tests/generic_views/views.py

```diff
@@ -307,3 +307,13 @@ def get_object(self, queryset=None):
 class ObjectDoesNotExistDetail(generic.DetailView):
     def get_queryset(self):
         return Book.does_not_exist.all()
+
+
+class LateValidationView(generic.FormView):
```

Comment:
@dudepare , sure, but I thought we should keep them separated in case the vanilla ContactView will be modified. Some sort of stability :).